cylc/uiserver/resolvers.py

```diff
@@ -15,10 +15,189 @@
 
 """GraphQL resolvers for use in data accessing and mutation of workflows."""
 
+import getpass
+import logging
+import os
+from shlex import quote
+from subprocess import Popen, PIPE, DEVNULL
+
 from cylc.flow.network.resolvers import BaseResolvers
 from cylc.flow.data_store_mgr import WORKFLOW
 
 
+DEBUG = True
```

Comment:
Shouldn't we use the `debug` argument from the `Play` mutation (from the schema)?

Reply:
Ah, sorry I didn't add any comments here to explain what this variable is for.

This is a response to the "no traceback" secure mode, I thought I would add a flag here to make it easier to implement this later.

There is a cylc-flow issue, which is slightly unhelpful as I think it's the UIS that this really applies to - https://github.com/cylc/cylc-flow/issues/3834